### ISSUE TITLE
Interface usage

### ISSUE THREAD

**Issue by caspervanzoest** [stuartpittaway/WebastoWBUSInterface#1]
Nice work!

I've created this circuit using the example & descriptions, 
However, I have a few questions on the usage,
since it is not (yet) working for me.
I am trying to debug what is wrong.

- Should te LED be always on or is it supposed to flash on either (or both?) TX / RX?
- The RX and TX are from the USB serial device point of view, or from the wbus point of view?
   (i.e. Transmit from vbus is recieve on USB)
- Are both 5V and 12V input voltages, or is either one of them output voltage?
- If I put 5v on the TX pin, should the wbus pin have a 12v output?
- if I put 12v on the Wbus pin, should the RX pin have 5v output?

I hope you can help me with answering the questions.
Thanks in advance.

**Comment by stuartpittaway** [comment]
Hi, I've not used this circuit for a long time so I don't remember what it should do!  I'll do still have the circuit board so I'll try and take a look tonight.

**Comment by caspervanzoest** [comment]
Thank you!
I am looking forward to your answer.

**Comment by stuartpittaway** [comment]
Okay, have you built the breadboard circuit or your own following the diagram here?

https://github.com/stuartpittaway/WebastoWBUSInterface/blob/master/Webasto%20WBus%20Arduino%20Serial%20Interface%20mk7_schem.png

There are 3 connections to the heater - WBUS, +12V and GROUND.

There are 4 connections to the FTDI (or computer interface) +5V, GROUND and RX/TX

The LED should flash when data is sent or received.

The circuit is a basic 5v to 12v logic level shifter which converts two wire TX/RX into a single wire WBUS signal.

**Comment by caspervanzoest** [comment]
Sounds good, thanks!
I guess I have used a wrong transistor in the circuit, since I did not have a correct one.
I've ordered the specified ones today, so hopefully I can show you the result tomorrow.

Since I am building this for somebody else, and I do not have a Webasto device laying around,
I assume I can test the hardware by putting 5v on the TX pin, and see if the wbus pin got a 12v output?
As well as putting 12v on the w-bus, and seeing if the RX pin is 5v positive, right?
